docs/contributing.md

```diff
@@ -128,6 +128,26 @@ Fix: #902
 
 For more details, please see [Link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
 
+### Modify PRs
+
+Inevitably, your PR may need to be revised. Please **re-use** the same PR when the code needs changes. Don't close 
+the PR and open a new one.
+
+Here is a possible example:
+- Modify the code in your local.
+- Modify this commit.  
+```shell
+git commit --amend
+```
```

Comment:
I think this is not a recommended method.

refer to https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#about-pull-requests

I think the general method is to create a new branch, submit your commit on the branch, and then create pull request. If you want to modify it, just submit a new commit to the branch, the pr page will automatically refresh.

Reply:
I think `git --amend`  can be used as advanced usage, but not general usage

Reply:
@Selflocking appending new commits are generally not bad. But in pracice, the PR author usually forgot to obey the semantic-pr rule when they add new commits. So we recommend modifying the original commit